frontend/packages/shared/src/utils/featureToggleUtils.ts

```diff
@@ -103,5 +103,8 @@ const shouldPersistInSession = (): boolean => {
 // Add feature to session storage to persist the feature in the current session
 const addFeatureFlagToSessionStorage = (featureFlag: SupportedFeatureFlags): void => {
   const featureFlagsFromStorage = typedSessionStorage.getItem<string[]>(featureFlagKey) || [];
+
+  const featureFlagAlreadyExist = featureFlagsFromStorage.includes(featureFlag);
+  if (featureFlagAlreadyExist) return;
```

Comment:
This did not have anything with the failing Cypress to do. However, I noticed that we saved the same value multiple times within the storage. Added a check to verify that is not already stored.